src/pudl_rmi/coordinate.py

```diff
@@ -0,0 +1,247 @@
+                output of the EIA plant part list and generate a new version of
+                the depreciaiton to FERC1 output. Default is False.
+        """
+        # if any of the clobbers are on, we want to regenerate the main output
+        clobber_any = any([clobber, clobber_ppl])
+        if not pudl_rmi.FILE_PATH_FERC1_EIA.is_file() or clobber_any:
+            logger.info(
+                "FERC to EIA granular connection not found at "
+                f"{pudl_rmi.FILE_PATH_FERC1_EIA}... Generating a new output."
+            )
+            connects_ferc1_eia = pudl_rmi.connect_ferc1_to_eia.execute(
+                self.pudl_out,
+                self.get_plant_part_list(clobber=clobber_ppl)
+            )
+            # export
+            connects_ferc1_eia.to_pickle(pudl_rmi.FILE_PATH_FERC1_EIA)
+        else:
+            logger.info(
+                "Reading the FERC1 to EIA connection from "
+                f"{pudl_rmi.FILE_PATH_FERC1_EIA}"
+            )
+            connects_ferc1_eia = pd.read_pickle(pudl_rmi.FILE_PATH_FERC1_EIA)
+        return connects_ferc1_eia
+
+    def get_deprish_to_ferc1(
+        self,
+        clobber=False,
+        clobber_ppl=False,
+        clobber_de=False,
+        clobber_fe=False
+    ):
+        """
+        Generate or grab a connection between deprecaiton data and FERC1.
+
+        Either generate or grab an on-disk cached version of the connection
+        between the depreciation study data and FERC1.
+
+        Args:
+            clobber (boolean): Generate and cache a new output even if it
+                exists on disk. This does not necessarily regenerate the
+                interim inputs - see other `clobber_` arguments. Default is
+                False.
+            clobber_ppl (boolean): Generate and cache a new interim
+                output of the EIA plant part list and generate a new version of
+                the depreciaiton to FERC1 output. Default is False.
+            clobber_de (boolean): Generate and cache a new interim output of
+                the connection between EIA and depreciation data and generate
+                a new version of the depreciaiton to FERC1 output. Default is
+                False.
+            clobber_fe (boolean): Generate and cache a new interim output of
+                the connection between FERC and EIA and generate a new version
+                of the depreciaiton to FERC1 output. Default is False.
+
+        Returns:
+            pandas.DataFrame: depreciation study data connected to FERC1 data
+            which has been scaled down or aggregated to the level of reporting
+            in the depreciaiton studies.
+        """
+        # if any of the clobbers are on, we want to regenerate the main output
+        clobber_any = any([clobber, clobber_ppl, clobber_de, clobber_fe])
+        if not pudl_rmi.FILE_PATH_DEPRISH_FERC1.is_file() or clobber_any:
```

Comment:
The same issue of this acting weird if the path exists but is not a file -- I think probably you want `exists()` but if it's a directory or other non-file I'm not sure just a straight overwrite will get the right behavior. You may need to remove it explicitly first. But I think the `Path` object has methods for this. But they're different for file/symlink vs. directory

https://docs.python.org/3/library/pathlib.html#pathlib.Path.unlink